otterdog/eclipse-daanse.jsonnet

```diff
@@ -42,7 +42,6 @@ orgs.newOrg('eclipse-daanse') {
       has_wiki: false,
       branch_protection_rules: [
         orgs.newBranchProtectionRule('main') {
-          required_approving_review_count: 1,
```

Comment:
if you remove this setting, the value from the default config will be taken into account, which specifies a value of 2.
From the title of your PR I would assume that you do not want any required approvals for now, so you should set this value to 0.

Reply:
thx will fix this

Reply:
@netomi fixes. thx